### PR TITLE
Stabilize traffic light - First part

### PR DESCRIPTION
**Pull request by hlloreda**
# Description

- Increased `InmantaBootLoader` timeout when everything is being shut down -> to make sure every component has enough time to be stopped without leaking resources (background task, ...)
- Remove sugar in logs to not hide teardown
- Add more logs when Scheduler tests are failing (check if other processes are interefering)

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )
- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s) (see [test-fixes](https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes) for more info)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -64,5 +64,7 @@ markers = [
 log_format = "%(asctime)s.%(msecs)03d %(levelname)s %(message)s"
 
 [tool.coverage.run]
+concurrency = ["multiprocessing"]
```
Comment (hlloreda) [inline pyproject.toml:67]: Needed by pytest-cov, see https://pytest-cov.readthedocs.io/en/latest/subprocess-support.html#if-you-use-multiprocessing for more information

File: tox.ini [hunk 2]
```diff
@@ -14,7 +14,7 @@ deps=
 extras=
     dataflow_graphic
 # Set the environment variable INMANTA_EXTRA_PYTEST_ARGS='--fast' to run in fast mode
-commands=timeout -s SIGINT {env:INMANTA_TIMEOUT_TESTSUITE:50}m py.test --log-level DEBUG --full-trace --cov=inmanta --cov-report term --cov-report xml --cov-config=pyproject.toml --junitxml=junit-{envname}.xml -vvv {env:INMANTA_EXTRA_PYTEST_ARGS:} --durations={env:INMANTA_TIMEOUT_TESTSUITE:50} tests
+commands=timeout -s SIGINT {env:INMANTA_TIMEOUT_TESTSUITE:50}m py.test --log-level DEBUG --full-trace -p no:sugar --cov=inmanta --cov-report term --cov-report xml --cov-config=pyproject.toml --junitxml=junit-{envname}.xml -vvv {env:INMANTA_EXTRA_PYTEST_ARGS:} --durations={env:INMANTA_TIMEOUT_TESTSUITE:50} tests
```
Comment (hlloreda) [inline tox.ini:17]: Should we remove pytest-sugar from the dependencies?

File: pyproject.toml (hunk 1 above)
Comment (wouterdb) [reply pyproject.toml:67]: I would add this as a comment in the file

File: tox.ini (hunk 2 above)
Comment (wouterdb) [reply tox.ini:17]: I would ask this on slack

**Review by wouterdb — CHANGES_REQUESTED** [review]

**Review by wouterdb — APPROVED** [review]

**Review by jptrindade — APPROVED** [review]

**Pull request closed without merging**